### ISSUE TITLE
Use positive instead of negative phrases

### ISSUE THREAD

**Issue by delucas** [kossnocorp/opensource.how#3]
I love this repo. Really... but... when I first read it, I thought it was very negative.
Don't do this, don't do that... is there room for a positive version of this guide?

Like...
**Don't be mean!** -> **Be nice!**
or **Don't demand!** -> **Ask politely!**

It's just an idea :)
Thanks for the effort of putting this guide together.

**Comment by ghostsquad** [comment]
I agree with this. I read the readme, and the first and only question I had was "what should I say instead?"

Some of the phrases that mentioned to stay away from are literally just asking for an update, a check in, anything that shows movement is being made.

One of the first and most valuable lessons I learned from being a parent was not to say "stop", unless there's imminent danger ahead. Instead, say "do this instead". It doesn't just work on kids, it's a human psychological thing. "Don't draw on the walls" turns into "draw on paper instead". Find the reason for the action, and help the user direct that energy the right way.

**Comment by kossnocorp** [comment]
Hey!

I understand and agree with you! When I shared this document on Reddit I got a lot of negative comments from people that were defensive. They were focused on defending their freedom to ask about updates, which is understandable, but that distracts from the more significant point.

I realize that focusing that much on dont's I miss the opportunity to promote better alternative behavior which is the whole point of this initiative.

However, while "Don't be mean!" and "Don't demand" are easier to turn into positives, it's a bit harder to rephrase "Don't be bossy!" and "Don't be noisy!" preserving the idea. Also, these headers are catchy and sparkle debate, which is also good.

Said that I open to suggestions on how I could rewrite this document. I'm open to hearing everything from ideas on how to rephrase the headers to an alternative structure.

**Comment by ghostsquad** [comment]
Just a refresher comment, I have plans to submit at least a PR for this. I like the idea of having etiquette like this be out in the open instead of an internal Confluence document (like what I currently have). So that I can say "other people think this is good etiquette too" - and maybe a little bit of "peer pressure" can be used to encourage the right behavior. 😄

**Comment by kossnocorp** [comment]
@ghostsquad looking forward!
